src/LercLib/include/Lerc_c_api.h

```diff
@@ -45,15 +45,18 @@ extern "C" {
 #define LERC_AT_LEAST_VERSION(maj,min,patch) \
     (LERC_VERSION_NUMBER >= LERC_COMPUTE_VERSION(maj,min,patch))
 
-#if defined(_MSC_VER)
-  #define LERCDLL_API __declspec(dllexport)
+#ifdef _WIN32

```

Comment:
Might want to change to

#if defined _WIN32 || defined __CYGWIN__

to cover all Windows possibilities once and for all